### ISSUE TITLE
Exception when examining an edit control inside a ListView (64bit Python, 32bit application)

### ISSUE THREAD

**Issue by hvbtup** [pywinauto/pywinauto#465]
I am examining an item under the mouse cursor like this:

                    mouse_pos = win32api.GetCursorPos()
                    print(f"Maus: {mouse_pos}")
                    # Top-Level-Window finden
                    hwnd = win32gui.WindowFromPoint(mouse_pos)
                    # Child-Window finden
                    tiefe = 0
                    while hwnd and tiefe <= 10:
                        print(f"handle={hwnd}")
                        relative_pos = win32gui.ScreenToClient(hwnd, mouse_pos)
                        print(f"relativ: {relative_pos}")
                        child = win32gui.ChildWindowFromPoint(hwnd, relative_pos)
                        if not child or child == hwnd:
                            break
                        tiefe += 1
                        hwnd = child
                    wrapped = app.window(handle=hwnd)
                    # hervorheben
                    wrapped.draw_outline()
                    # Properties dieses Fensters herausfinden
                    # und in der Hierarchie wieder bis zum Top-Level zurückgehen.
                    while hwnd:
                        wrapped = app.window(handle=hwnd)
                        spec_dict = {}
                        title = wrapped.window_text()
                        if title: 
                            spec_dict["title"] = title  # TODO Escapen
                        spec_dict["friendly_class_name"] = wrapped.friendly_class_name()
                        control_id = wrapped.control_id()
                        if control_id:
                            spec_dict["control_id"] = control_id
                        spec_text = []
                        for k,v in spec_dict.items():
                            if isinstance(v, int):
                               spec_text.append("%s = %s" % (k,v))
                            else:
                               spec_text.append("%s = '''%s'''" % (k,v))
                        path.append('.window(%s)' % (",".join(spec_text)))
                        hwnd = wrapped.parent()
                    # Umdrehen, damit das Top-Level-Fenster vorne steht.
                    path.reverse()
                    # I geeigneter Weise ausgeben
                    print("app\\\n" + "\\\n".join([("    %s" % elem) for elem in path]))
                    # etwas warten, damit der Anwender das erkannte Item sehen kann.
                    time.sleep(0.5)

(well, I am working on a helper for creating test scripts)

My Python is 64bit, the application I am examining is a 32bit Java SWT program.
Basically everything works fine.

But I get exceptions like this

    _, process_id = win32process.GetWindowThreadProcessId(int(handle))
    TypeError: int() argument must be a string, a bytes-like object or a number, not 'ListViewWrapper'

whenever the item under the mouse cursor is an Edit control inside a ListView or when the mouse cursor is over the ListView header line.

I could work around this by modifying a line inside common_controls.py ListViewWrapper.__init__ : 
Old:
            if is64bitprocess(self.process_id()) or not is_x64_Python():
New:
            if False:

But this is a work-around; the error seems to be that the processid in element_info is of the wrong type!?

**Comment by vasily-v-ryabov** [comment]
Hi @hvbtup the problem is in `int(handle)` where `handle` type is `ListViewWrapper`.

Just use the following:
```python
_, process_id = win32process.GetWindowThreadProcessId(handle.handle)
```
Of course `handle` should be renamed to wrapper probably. Also it can left as is:
```python
_, process_id = win32process.GetWindowThreadProcessId(handle.handle)
```
because `_as_parameter` attribute will convert it to `int` automatically when it's used as a function parameter. Just a small black magic spell of Python.

**Comment by hvbtup** [comment]
So I patched handle_props.py like this:

    def processid(handle):
        """Return the ID of process that controls this window"""
        if hasattr(handle, "handle"):
            _, process_id = win32process.GetWindowThreadProcessId(int(handle.handle))
        else:
            _, process_id = win32process.GetWindowThreadProcessId(int(handle))
        return process_id

Is this what you meant?

**Comment by airelil** [comment]
@hvbtup , something looks fishy here. Correct me if I wrong. The `hwnd` parameter expected to be a window handle, but in your code, on the second loop, you end up passing a wrapper object instead:
```python
        while hwnd:
            wrapped = app.window(handle=hwnd)
            spec_dict = {}
             title = wrapped.window_text()
             # ...
             # skip the rest 
             # ...
             hwnd = wrapped.parent()
```

**Comment by vasily-v-ryabov** [comment]
@hvbtup it depends what you pass in `handle` parameter of the function. This implementation might work for you.
